net-ftp/ncftp/ncftp-3.2.6-r4.ebuild

```diff
@@ -0,0 +1,56 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+inherit autotools toolchain-funcs
+
+DESCRIPTION="An extremely configurable ftp client"
+HOMEPAGE="https://www.ncftp.com/"
+SRC_URI="
+	https://ftp.mirrorservice.org/sites/ftp.${PN}.com/${PN}/${P}-src.tar.xz
+"
+
+LICENSE="Clarified-Artistic"
+SLOT="0"
+KEYWORDS="~alpha ~amd64 ~arm ~hppa ~ia64 ~mips ~ppc ~ppc64 ~s390 ~sparc ~x86 ~amd64-linux ~x86-linux ~ppc-macos ~x64-macos ~sparc-solaris ~x86-solaris"
+IUSE="pch"
+
+DEPEND="
+	sys-libs/ncurses:=
+"
+RDEPEND="
+	${DEPEND}
+"
+BDEPEND="virtual/pkgconfig"
+
+PATCHES=(
+	"${FILESDIR}"/${PN}-3.2.6-fno-common.patch
+)
+
+src_prepare() {
+	default
+
+	sed -i -e '/^AR=/d' autoconf_local/aclocal.m4 || die
+	# 727774
+	sed -i -e 's/STRIP=".*"/STRIP=":"/' autoconf_local/aclocal.m4 || die
+
+	AT_M4DIR=autoconf_local/ eautoreconf
+}
+
+src_configure() {
+	tc-export AR CC
+	LC_ALL="C" \
+	LIBS="$( $(tc-getPKG_CONFIG) --libs ncurses)" \
+		econf \
+		$(use_enable pch precomp) \
+		--disable-ccdv \
+		--disable-universal
+}
+
+src_install() {
+	default
+
+	DOCS=( README.txt doc/*.txt )
```

Comment:
Sometimes  a glob in `DOCS` could cause issues,  see also https://github.com/gentoo/gentoo/pull/23668#issuecomment-1010143030. 

I recommend to use `dodoc` and `dohtml`, potentially with `-r`, instead.

Reply:
`DOCS` and `HTML_DOCS` with globbing cause problems if declared at global scope, but used inside a function they work fine.

But passing `-r` to `einstalldocs` does nothing, so I should change this anyway.

`dohtml` is banned in EAPI 7, and the suggestion [here](https://devmanual.gentoo.org/function-reference/install-functions/) is to switch to `einstalldocs`, that's why I tried this.  I also tried declaring `DOCS` and `HTML_DOCS` at global level without globbing, but I didn't like the result.

So I can either remove `-r` or simply revert to `docinto`/`dodoc` as in `ncftp-3.2.6-r3.ebuild`.  What do you think is better?

Reply:
I think

```
dodoc README.txt doc/*.txt
docinto html
dodoc doc/html/*.html
```

is just fine. One could maybe consider replacing the final `dodoc` with `dodoc -r doc/html`.

Reply:
Done.  Thank you